crates/ide-assists/src/handlers/desugar_doc_comment.rs

```diff
@@ -57,25 +57,30 @@ pub(crate) fn desugar_doc_comment(acc: &mut Assists, ctx: &AssistContext<'_>) ->
         }
     };
 
+    let text = match comments {
+        Either::Left(comment) => {
+            let text = comment.text();
+            text[comment.prefix().len()..(text.len() - "*/".len())]
+                .trim()
+                .lines()
+                .map(|l| l.strip_prefix(&indentation).unwrap_or(l))
+                .join("\n")
+        }
+        Either::Right(comments) => {
+            let mut cms: Vec<String> = Vec::new();
```

Comment:
I think you can do something like `let cms = comments.into_iter().map(|cm| line_comment_text(IndentLevel(0), cm)).collect::<Result<Vec<_>, _>()?`.